radis/tools/read_wav_index.py

```diff
@@ -0,0 +1,93 @@
+import bisect
+import json
+import os
+
+from radis.misc.utils import getProjectRoot
+
+# Load index
+wav_index_path = os.path.join(getProjectRoot(), "db", "wav_index.json")
+with open(wav_index_path, "r") as f:
+    wav_index = json.load(f)
+
+wavno_keys = sorted(float(k) for k in wav_index.keys())
```

Comment:
map(float, wav_index) would be a bit more elegant here

Reply:
I guess you do this because the dict you load is str:str? please add that in the comments